modules/install-telegraf/install-telegraf

```diff
@@ -98,7 +109,11 @@ function install_telegraf {
     exit 1
   fi
 
+  # Disable the installed systemd service to prevent it from starting up on boot
```

Comment:
Mention that the run script will be used to start the service after configuring it properly.